src/main/java/core/basesyntax/strategy/BalanceOperationHandler.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.strategy;
+
+import static core.basesyntax.db.Storage.fruitData;
+
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperationHandler implements OperationHandler {
+
+    @Override
+    public void process(FruitTransaction fruitTransaction) {
+        if (fruitData.containsKey(fruitTransaction.getFruit())) {
+            Integer fruitAmountInfo = fruitData.get(fruitTransaction.getFruit());
+            fruitAmountInfo += fruitTransaction.getAmount();
+            fruitData.replace(fruitTransaction.getFruit(),fruitAmountInfo);
+        } else {

```

Comment:
see `getOrDefault()` or `merge()` methods in Map
Using one of them you can improve your code.